frontend/app/visualizations/chart/index.js

```diff
@@ -66,6 +65,11 @@ function ChartEditor(ColorPalette) {
         scatter: { name: 'Scatter', icon: 'circle-o' },
       };
 
+      console.log(clientConfig);
+      if (clientConfig.allowCustomJSVisualizations) {
```

Comment:
Need to use this in the render function too, so that someone clever won't manually submit this type.